autonomy/chain/mint.py

```diff
@@ -53,6 +53,28 @@ def transact(ledger_api: LedgerApi, crypto: Crypto, tx: Dict) -> Dict:
     return ledger_api.get_transaction_receipt(tx_digest=tx_digest)
 
 
+def sort_service_dependency_metadata(
+    agent_ids: List[int],
+    number_of_slots_per_agents: List[int],
+    cost_of_bond_per_agent: List[int],
+) -> Tuple[List[int], ...]:
+    """Sort service dependencies and their respective parameters"""
```

Comment:
To address https://github.com/valory-xyz/open-autonomy/pull/1726#discussion_r1066996776